README.md

```diff
@@ -42,7 +42,6 @@ flavours of Linux](https://snapcraft.io/microk8s). Perfect for:
   - Ingress, DNS, Dashboard, Clustering
   - Automatic updates to the latest Kubernetes version
   - GPGPU bindings for AI/ML
-  - Kubeflow!
```

Comment:
I would not mind having this be a link pointing to the [Charmed Kubeflow docs](https://charmed-kubeflow.io/).